### PR TITLE
clean up test code

### PR DESCRIPTION
**Pull request by carlory**
**What type of PR is this?**

/kind cleanup
/kind testing

**What this PR does / why we need it**:

Use log functions of core e2e framework instead of e2elog sub package

```
test/utils/crd/crd_util.go
test/e2e/gke_local_ssd.go
test/e2e/examples.go
test/e2e_node/*.go
test/e2e/node/*.go
```

**Which issue(s) this PR fixes**:

refer #81427

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

**Comment by k-toyoda-pi** [comment]
/lgtm
But, conflicting file is appeared.

**Comment by oomichi** [comment]
I am guessing old version of this changed unrelated files then it has multiple unrelated labels.
We need to remove the labels for reviewing.

/remove-area apiserver
/remove-area code-generation
/remove-kind api-change
/remove-sig api-machinery
/remove-sig apps
/remove-sig auth
/remove-sig autoscalling
/remove-sig lifecycle
/remove-sig network
/remove-sig storage

File: test/e2e/node/events.go [hunk 1]
```diff
@@ -20,16 +20,14 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/onsi/ginkgo"
```
Comment (oomichi) [inline test/e2e/node/events.go]: This is not necessary to be moved in this PR.
The original sort order is common in e2e tests.
That means we need to keep like
```
...
	"k8s.io/kubernetes/test/e2e/framework"

	"github.com/onsi/ginkgo"
```
with a single blank line between framework and ginkgo.

File: test/e2e/node/pods.go [hunk 2]
```diff
@@ -25,6 +25,7 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/onsi/ginkgo"
```
Comment (oomichi) [inline test/e2e/node/pods.go]: ditto

File: test/e2e/node/ttlafterfinished.go [hunk 3]
```diff
@@ -19,15 +19,13 @@ package node
 import (
 	"time"
 
+	"github.com/onsi/ginkgo"
```
Comment (oomichi) [inline test/e2e/node/ttlafterfinished.go]: ditto

File: test/e2e_node/dynamic_kubelet_config_test.go [hunk 4]
```diff
@@ -33,9 +33,8 @@ import (
 	controller "k8s.io/kubernetes/pkg/kubelet/kubeletconfig"
 	"k8s.io/kubernetes/pkg/kubelet/kubeletconfig/status"
 	"k8s.io/kubernetes/pkg/kubelet/metrics"
-	frameworkmetrics "k8s.io/kubernetes/test/e2e/framework/metrics"
-
 	"k8s.io/kubernetes/test/e2e/framework"
+	frameworkmetrics "k8s.io/kubernetes/test/e2e/framework/metrics"
```
Comment (oomichi) [inline test/e2e_node/dynamic_kubelet_config_test.go]: We don't need to change here in this PR at all.

File: test/e2e_node/environment/conformance.go [hunk 5]
```diff
@@ -19,17 +19,16 @@ limitations under the License.
...
 	"io/ioutil"
 	"net"
+	"os"
 	"os/exec"
 	"regexp"
 	"strings"
 
-	"errors"
-	"os"
-
```
Comment (oomichi) [inline test/e2e_node/environment/conformance.go]: ditto: We don't need to change this file at all for the PR scope.

File: test/e2e_node/hugepages_test.go [hunk 6]
```diff
@@ -27,10 +27,8 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/uuid"
-
```
Comment (oomichi) [inline test/e2e_node/hugepages_test.go]: I think the original blank line here is for separating k8s.io/kubernetes repo and other k8s.io repos, so we don't need to remove this blank line.

File: test/e2e_node/pids_test.go [hunk 7]
```diff
@@ -24,11 +24,9 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/uuid"
-
```
Comment (oomichi) [inline test/e2e_node/pids_test.go]: ditto: unnecessary to remove a blank line.

File: test/e2e_node/pods_container_manager_test.go [hunk 8]
```diff
@@ -23,13 +23,13 @@ import (
...
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/klog"
 	"k8s.io/kubernetes/pkg/kubelet/cm"
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
 	imageutils "k8s.io/kubernetes/test/utils/image"
 
 	"github.com/onsi/ginkgo"
-	"k8s.io/klog"
```
Comment (oomichi) [inline test/e2e_node/pods_container_manager_test.go]: We don't need to change this file at all.

File: test/e2e_node/remote/node_conformance.go [hunk 9]
```diff
@@ -26,7 +26,6 @@ import (
 	"time"
 
 	"k8s.io/klog"
-
```
Comment (oomichi) [inline test/e2e_node/remote/node_conformance.go]: ditto: don't change..

File: test/e2e_node/remote/node_e2e.go [hunk 10]
```diff
@@ -25,7 +25,6 @@ import (
 	"time"
 
 	"k8s.io/klog"
-
```
Comment (oomichi) [inline test/e2e_node/remote/node_e2e.go]: ditto

File: test/e2e_node/runner/local/run_local.go [hunk 11]
```diff
@@ -24,11 +24,10 @@ import (
 	"path/filepath"
 	"strings"
 
+	"k8s.io/klog"
 	"k8s.io/kubernetes/test/e2e_node/builder"
 	"k8s.io/kubernetes/test/e2e_node/system"
 	"k8s.io/kubernetes/test/utils"
-
-	"k8s.io/klog"
```
Comment (oomichi) [inline test/e2e_node/runner/local/run_local.go]: ditto: don't change unrelated file.

File: test/e2e_node/services/internal_services.go [hunk 12]
```diff
@@ -22,9 +22,8 @@ import (
 
 	etcd3testing "k8s.io/apiserver/pkg/storage/etcd3/testing"
 	"k8s.io/apiserver/pkg/storage/storagebackend"
-	"k8s.io/kubernetes/test/e2e/framework"
-
 	"k8s.io/klog"
+	"k8s.io/kubernetes/test/e2e/framework"
```
Comment (oomichi) [inline test/e2e_node/services/internal_services.go]: ditto

File: test/e2e_node/services/kubelet.go [hunk 13]
```diff
@@ -27,11 +27,11 @@ import (
 	"time"
 
 	"github.com/spf13/pflag"
-	"k8s.io/klog"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	cliflag "k8s.io/component-base/cli/flag"
+	"k8s.io/klog"
```
Comment (oomichi) [inline test/e2e_node/services/kubelet.go]: ditto

File: test/e2e_node/services/server.go [hunk 14]
```diff
@@ -30,7 +30,6 @@ import (
 	"time"
 
 	"k8s.io/klog"
-
```
Comment (oomichi) [inline test/e2e_node/services/server.go]: ditto

File: test/e2e_node/services/services.go [hunk 15]
```diff
@@ -24,9 +24,8 @@ import (
 	"path"
 	"testing"
 
-	"k8s.io/klog"
-
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/klog"
```
Comment (oomichi) [inline test/e2e_node/services/services.go]: ditto

File: test/e2e_node/volume_manager_test.go [hunk 16]
```diff
@@ -25,8 +26,6 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
 
-	"fmt"
```
Comment (oomichi) [inline test/e2e_node/volume_manager_test.go]: ditto

**Review by oomichi — CHANGES_REQUESTED** [review]
Please don't change unrelated files.

**Comment by carlory** [comment]
removed unrelated changes

**Comment by oomichi** [comment]
We need to replace e2e_log in test/e2e_node/startup_probe_test.go also as
```
compile: missing strict dependencies:
	/b/f/w/test/e2e_node/startup_probe_test.go: import of "k8s.io/kubernetes/test/e2e/framework/log"
```
according to https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/81974/pull-kubernetes-bazel-build/1168019699871518721/build-log.txt

**Comment by carlory** [comment]
@oomichi thanks for your guide. It's fixed!

**Review by oomichi — COMMENTED** [review]
Thanks for updating

/lgtm
/approve

**Pull request merged**